<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setupModularSpeeds</class>
 <widget class="QDialog" name="setupModularSpeeds">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Set modular speeds</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sewing machine</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pinch roller top</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pinch roller bottom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Puller wheel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_machine_1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_machine_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_machine_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_machine_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Area 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Area 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Area 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Area 4</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pinch_top_1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pinch_top_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pinch_top_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pinch_top_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pinch_bottom_1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pinch_bottom_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pinch_bottom_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pinch_bottom_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_puller_1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_puller_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_puller_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_puller_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_back_puller_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_back_puller_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back puller wheel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_back_puller_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_back_puller_1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
